editor/plugins/version_control_editor_plugin.cpp

```diff
@@ -133,77 +140,183 @@ void VersionControlEditorPlugin::_initialize_vcs() {
 
 	ERR_FAIL_COND_MSG(!EditorVCSInterface::get_singleton()->initialize(res_dir), "VCS was not initialized");
 
+	EditorVCSInterface::get_singleton()->set_up_credentials(set_up_username->get_text(), set_up_password->get_text());
```

Comment:
I think we should handle the case if the user does not put anything as username or password. Maybe raise  a warning or a red coloured text if we find empty username and password